### PR TITLE
Updated to work with the latest on master of ofxMSAInteractiveObject. Example project now works

### PR DESCRIPTION
**Pull request by doukasd**
Hi, just got the latest form master from your (wonderful) project and from the dependencies and the "ofxMSAInteractiveObject" seems to have been updated and does not work out of the box, in particular due to the following commits:

https://github.com/memo/ofxMSAInteractiveObject/commit/c8b66347df047ddfdc572e5f8ef94e1bfc6990e5#diff-a41973565e0f3559262fa1030eb1cfde
https://github.com/memo/ofxMSAInteractiveObject/commit/c698272643d2fb60ca1c3d38831af5164c975369#diff-a41973565e0f3559262fa1030eb1cfde
https://github.com/memo/ofxMSAInteractiveObject/commit/414b13fc3efb3d7af0bdb5cf8f091bf5afa38533#diff-a41973565e0f3559262fa1030eb1cfde

In a nutshell the changes are:
- updated the setPos() to setPosition().
- compensated for the lack of the killMe() method.
- removed the deprecated isMouseDown() as I don't see how it is needed (I might be wrong, though it works fine).
- added a check in the snapIt() method as the app could crash if a vertex was selected, then the parent shape selected and then an arrow key pressed.

Please review as I might have missed something.

### CODE REVIEW

**Comment by smallfly** [comment]
Thanks!

**Comment by smallfly** [comment]
Please let me know what you do with this add on. I would love to see what people do with it.

Cheers,

Hugues

**Comment by doukasd** [comment]
I'm still evaluating this but it looks very helpful for a live Kinect tracking + Projection dance event I'm working on. If I put it to good use i'll let you know and I'll give it a star ;) Thanks!

**Comment by smallfly** [comment]
Great! Do not hesitate if you need something. And keep the pull requests coming ;)
